.env

```diff
@@ -8,7 +8,7 @@ PROJECT_NAME="TAD"
 # TAD backend
 BACKEND_CORS_ORIGINS="http://localhost,https://localhost,http://127.0.0.1,https://127.0.0.1"
 SECRET_KEY=changethis
-APP_DATABASE_SCHEME="postgresql"
```

Comment:
this should stay postgresql because else the compose.yaml does not work as desired